indigo_api/models/works.py

```diff
@@ -170,6 +170,19 @@ def repeal(self):
     def place(self):
         return self.locality or self.country
 
+    @property
+    def amended(self):
```

Comment:
`.all()` goes to the database and loads all the data, which is very expensive. Instead, use `.exists()`.

Since this method is "expensive" (ie. it goes to the database, it's not quick), don't make it a `@property`. A property should be something that is cheap to call. By making it a method, we're indicating that it's expensive to call.

Reply:
https://softwareengineering.stackexchange.com/questions/367161/when-is-it-not-pythonic-to-use-properties-instead-of-a-method-that-doesnt-take?rq=1

Reply:
thanks! if I've understood you, if I use `.exists()` instead I can leave it as a property?